.pre-commit-config.yaml

```
@@ -43,7 +43,13 @@ repos:
     hooks:
     -   id: mypy
 
--   repo: https://github.com/pre-commit/mirrors-pylint
-    rev: v2.3.1
+-   repo: local
     hooks:
     -   id: pylint
+        language: system
+        name: PyLint
+        files: \.py$
+        entry: python3 -m pylint
+        args:
+        - msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
```

The first dash is just for YAML to detect a sequence item. The following contents is a text node `msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}`.

These values go to the command which `pre-commit` will construct. In this case, it would be 
```console
python3 -m pylint 'msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}' ...
```
which is wrong. Normally, CLI tools accept arguments prefixed with hyphens. Double dash for a `--long-version` and a single dash for its `-sv`.
So you need to produce the command as follows 
```console
python3 -m pylint '--msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}' ...
```

Which means that you have to explicitly type all the necessary chars:
```suggestion
        - --msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
```